core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/repository/RevisionDocument.java

```diff
@@ -54,6 +56,14 @@ public static Expression idPrefix(String idPrefix) {
 		public static Expression idRegex(String idRegex) {
 			return regexp(Fields.ID, idRegex);
 		}
+		
+		public static Expression anyFieldValuePresent(final String field, final String value, final int minShouldMatch) {
```

Comment:
These are just proxies over the `Expressions` class static method, they do not add actual value to the `RevisionDocument`, please remove these and use the `Expressions` methods directly instead.